routes/api/resources.js

```diff
@@ -9,10 +9,14 @@ const helpers = require('../helpers');
 const router = express.Router();
 
 router.get('/', async (req, res) => {
-  const records = await models.Resource.findAll();
-
+  const options = {};
+  if (req.query.categoryId) {
+    options.where = {
+      
```

Comment:
Here is where you can specify a filter for the findAll() query. Look at the corresponding test to see how you specify a Category in the request.

Here's the documentation for the WHERE clause in queries:

https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#applying-where-clauses